src/Microsoft.AspNetCore.Razor.Evolution/Legacy/CSharpCodeParser.cs

```diff
@@ -31,6 +31,21 @@ internal class CSharpCodeParser : TokenizerBackedParser<CSharpTokenizer, CSharpS
                 .AddType()
                 .Build();
 
+        internal static readonly DirectiveDescriptor AddTagHelperDirectiveDescriptor =
+            DirectiveDescriptorBuilder
+                .Create(SyntaxConstants.CSharp.AddTagHelperKeyword)
+                .Build();
```

Comment:
I agree with @NTaylorMullen - if we're going to use these descriptors to describe the directives, we should make them accurate.